runtime/oti/util_api.h

```diff
@@ -1753,6 +1753,16 @@ isSameOrSuperInterfaceOf(J9Class *superInterface, J9Class *baseInterface);
 J9VMThread *
 getVMThreadFromOMRThread(J9JavaVM *vm, omrthread_t omrthread);
 
+/**
+ * @brief Initialize the currentOSStackFree field in the J9VMThread. Must be called by
+ *	running thread, not thread creating the new vmThread.
+ * @param vmThread vmThread token
```

Comment:
if this must always be the current thread, I'd prefer the parameter name reflect that (the parameter to the tracepoint must be the current thread).